### PR TITLE
Proposal: add podman_container_copy module

### PR DESCRIPTION
**Pull request by kubealex**
As the title suggests, this PR covers a draft proposal of a module for podman cp command.

It basically reflects the podman cp usage, allowing host -> container copy (default) or container -> host (configurable via a parameter).

Attached are also the tests for both flows (host->container, container->host)

### CODE REVIEW

**Comment by sshnaidm** [comment]
@kubealex awesome,thanks for contribution! I'll review it asap. For running tests in this patch it's better to include all CI bits from here: https://github.com/containers/ansible-podman-collections/pull/531 Just replace "podman_lovely" with "podman_container_copy"

File: plugins/modules/podman_container_copy.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    container: apache
+    from_container: True
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def copy_file(module, executable, src, dest, container, from_container, archive, overwrite):
+    if from_container:
+        command = [executable, 'cp', '{}:{}'.format(container, src), dest]
```
Comment (sshnaidm) [inline plugins/modules/podman_container_copy.py:78]: Ansible sanity complains in those two lines:
```
ERROR: plugins/modules/podman_container_copy.py:78:37: ansible-format-automatic-specification: Format string contains automatic field numbering specification
ERROR: plugins/modules/podman_container_copy.py:80:42: ansible-format-automatic-specification: Format string contains automatic field numbering specification
```

Comment (kubealex) [reply plugins/modules/podman_container_copy.py]: Fixed, thank you!

**Comment by kubealex** [comment]
thank you @sshnaidm! 
I just added the CI part, fixed the issue and squashed the commits to a single one.

File: .github/workflows/podman_container_copy.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
+name: Podman lovely module
```
Comment (sshnaidm) [inline .github/workflows/podman_container_copy.yml:1]: Please update the name

File: plugins/modules/podman_container_copy.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+module: podman_container_copy
+author:
+  - Alessandro Rossi (@kubealex)
+short_description: Copy file into running container
```
Comment (sshnaidm) [inline plugins/modules/podman_container_copy.py:13]:
```suggestion
short_description: Copy file into running container or from it
```

File: plugins/modules/podman_container_copy.py [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+        argument_spec=dict(
+            executable=dict(type='str', default='podman'),
+            src=dict(type='str', required=True),
+            dest=dict(type='str', required=True),
+            container=dict(type='str', required=True),
+            from_container=dict(type='bool', required=False, default=False),
+            archive=dict(type='bool', required=False, default=True),
+            overwrite=dict(type='bool', required=False, default=False)
+        ),
+        supports_check_mode=True,
```
Comment (sshnaidm) [inline plugins/modules/podman_container_copy.py:108]: I don't think it supports check_mode it in current revision, isn't it?

File: .github/workflows/podman_container_copy.yml [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+      fail-fast: false
+      matrix:
+        ansible-version:
+          - ansible<2.10
+          # - git+https://github.com/ansible/ansible.git@stable-2.11
+          - git+https://github.com/ansible/ansible.git@devel
+        os:
+          - ubuntu-22.04
+        python-version:
+          - 3.9
```
Comment (sshnaidm) [inline .github/workflows/podman_container_copy.yml:43]:
```suggestion
          - 3.11
```

**Review by sshnaidm — CHANGES_REQUESTED** [review]
A few comments inside

File: plugins/modules/podman_container_copy.py (hunk 4 above)
Comment (kubealex) [reply plugins/modules/podman_container_copy.py:108]: correct, let me change it!

File: plugins/modules/podman_container_copy.py (hunk 3 above)
Comment (kubealex) [reply plugins/modules/podman_container_copy.py:13]: I changed it to "Copy file to/from a container" because it doesn't require to be running.

**Review by sshnaidm — APPROVED** [review]
Great, thanks for this module!

**Comment by kubealex** [comment]
Thank you @sshnaidm  :)

**Pull request merged**